--- a/data/im.bernard.Nostalgia.appdata.xml.in	2018-12-01 14:59:10.000000000 +0100
+++ b/data/im.bernard.Nostalgia.appdata.xml.in	2019-09-13 23:15:55.767531891 +0200
@@ -25,6 +25,9 @@
             <caption>App Window</caption>
         </screenshot>
     </screenshots>
+    <releases>
+        <release version="0.3" date="2018-12-01" />
+    </releases>
     <url type="homepage">https://gitlab.gnome.org/bertob/nostalgia</url>
     <url type="bugtracker">https://gitlab.gnome.org/bertob/nostalgia/issues</url>
     <update_contact>tbernard@gnome.org</update_contact>
